includes/Story_Post_Type.php

```diff
@@ -269,11 +280,17 @@ public static function admin_enqueue_scripts( $hook ) {
 
 		$post             = get_post();
 		$story_id         = ( $post ) ? $post->ID : null;
-		$post_type_object = get_post_type_object( self::POST_TYPE_SLUG );
-		$rest_base        = ! empty( $post_type_object->rest_base ) ? $post_type_object->rest_base : $post_type_object->name;
 		$post_thumbnails  = get_theme_support( 'post-thumbnails' );
+		$rest_base        = self::POST_TYPE_SLUG;
```

Comment:
Not sure what issue we are fixing here...

Reply:
`$post_type_object` could be `null` in theory; PHPStan was flagging this.

Reply:
We have bigger problems here if `$post_type_object` isn't not an instance of `WP_Post_Type` here. Instead of trying to guess the rest base here, we should error out, it is likely it the post type is not setup here, then the rest api call will fail either way.

Reply:
What would "error out" mean here in your opinion?

Reply:
I mean, this is just about adding some safeguard to prevent null pointer exceptions. Throwing errors here wouldn't be nice, that's why this tries to prevent crashes.

Reply:
Just add a return.

Reply:
And completely break the editor because of that? 🤔